### PR TITLE
prep for registering, adding Travis-CI stuff

### PR DESCRIPTION
**Pull request by dehann**
Hey @GearsAD , you want to go [Travis with this link](https://travis-ci.org/profile/GearsAD), and activate build for CloudGraphs.jl there.

This pull request updates your ```README.md```, ```REQUIRE``` files and introduces the ```.travis.yml``` file.

### CODE REVIEW

**Comment by dehann** [comment]
oh wait I made a mistake, will fix now

**Comment by dehann** [comment]
Okay, that should be right.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,43 @@
 ![GitHub Logo](/logo.png)
 
 # CloudGraphs.jl
-Repo for the CloudGraphs project, ongoing project on http://semisortedblog.wordpress.com, check it out! :) 
+
+[![Build Status][build-img]][build-url]
+[![CloudGraphs][cg-badge-v0.5]][cg-pkg-v0.5]
+[![CloudGraphs][cg-badge-v0.6]][cg-pkg-v0.6]
```
Comment (dehann) [inline README.md]: The bad printing on the rendered readme should clear once the package shows up in pkg.julialang.org.

Comment (GearsAD) [inline README.md]: Looks good!

**Pull request merged**